Classes/Service/ExportService.php

```diff
@@ -11,17 +11,31 @@
 class ExportService implements SingletonInterface
 {
     /**
-     * @var PAGEmachine\Ats\Domain\Repository\ApplicationRepository
-     * @inject
+     * @var ApplicationRepository $applicationRepository
      */
     protected $applicationRepository;
 
     /**
-     * @var PAGEmachine\Ats\Domain\Repository\JobRepository
-     * @inject
+     * @var JobRepository $jobRepository
      */
     protected $jobRepository;
 
+    /**
+     * @param ApplicationRepository $applicationRepository
+     */
+    public function injectApplicationRepository(ApplicationRepository $applicationRepository)
```

Comment:
The import for `ApplicationRepository` is missing.

Reply:
Added.